src/Microsoft.Health.Fhir.Api/Features/SMART/SmartClinicalScopesMiddleware.cs

```diff
@@ -112,7 +112,7 @@ public SmartClinicalScopesMiddleware(RequestDelegate next, ILogger<SmartClinical
 
                                 fhirRequestContext.AccessControlContext.AllowedResourceActions.Add(new ScopeRestriction(resource, permittedDataActions, id));
 
-                                _logger.LogInformation("Resource and permitted data actions {Resource} {PermittedDataActions} ", resource, permittedDataActions);
+                                scopeRestrictions.Append($" ( {resource}-{permittedDataActions} ) ");
```

Comment:
Could you verify the spacing, it seems it will add two spaces as there is a space in the beginning and space at the end too.